website/templates/prereg_landing_page.mako

```diff
@@ -0,0 +1,141 @@
+<%inherit file="base.mako"/>
+
+<%def name="title()">OSF Prereg Challenge</%def>
+
+<%def name="stylesheets()">
+    ${ parent.stylesheets() }
+    <link rel="stylesheet" href="/static/css/prereg.css">
+</%def>
+
+<%def name="javascript_bottom()">
+    ${ parent.javascript_bottom() }
+    <script src=${"/static/public/js/prereg-landing-page.js" | webpack_asset}></script>
+</%def>
+
+<%def name="newPrereg(size=None)">
+<% size = size or '' %>
+<div id="newPrereg${size}" class="prereg-new-prereg p-md osf-box box-round clearfix m-b-lg" style="display:none">
+  <p>Please provide a title for your project: </p>
+  <input type="text" class="new-project-title form-control" placeholder="Title">
+  <button type="submit" id="newProject${size}" class="btn btn-primary pull-right m-t-md">Continue <i class="fa fa-angle-right"></i></button>
+</div>
+</%def>
+
+<%def name="existingPrereg(size=None)">
+<% size = size or '' %>
+<div id="existingPrereg${size}" class="prereg-exisitng-prereg p-md osf-box box-round clearfix m-b-lg" style="display:none">
+  <p>Go to an existing preregistration:</p>
+  <form>
+    <osf-draft-registrations-search
+       params="placeholder: 'Type to search for a project',
+               data: '/api/v1/prereg/draft_registrations/',
```

Comment:
The `web_url_for` helper is directly available in the template context; there's no need to hard-code URLs here.